### PR TITLE
Handle Result<> and the closing

### PR DESCRIPTION
**Pull request by akhansari**

### CODE REVIEW

File: EsBankAccount.Tests/Startup/BankAccountClientTests.fs [hunk 1]
```diff
@@ -13,8 +13,8 @@ let fakeAccountId () = string Guid.Empty
 let ``Should deposit and then withdraw`` () =
     async {
         let accountId = fakeAccountId ()
-        do! deposit accountId 10m
-        do! withdraw accountId 5m
-        let! transac = ReadModelClient.transactionsOf accountId
-        transac.Length =! 2
+        do! deposit accountId 10m |> Async.Ignore
```
Comment (bartelink) [inline EsBankAccount.Tests/Startup/BankAccountClientTests.fs:16]: consider `let! _view = deposit accountId 10m` as a way to document what you are ignoring / the intent

File: EsBankAccount.Tests/Startup/BankAccountClientTests.fs [hunk 2]
```diff
@@ -13,8 +13,8 @@ let fakeAccountId () = string Guid.Empty
...
+        do! withdraw accountId 5m |> Async.Ignore
+        let! account = ReadModelClient.getAccount accountId
```
Comment (bartelink) [inline EsBankAccount.Tests/Startup/BankAccountClientTests.fs:18]: Even if you are 'getting' it out of a cache, I find the term `read` is more indicative of the level of work involved

File: EsBankAccount.Tests/Startup/BankAccountClientTests.fs (hunk 1 above)
Comment (akhansari) [reply EsBankAccount.Tests/Startup/BankAccountClientTests.fs:16]: Interesting yes. I'll create a new review branch for this. Thanks Ruben 🙏

Comment (bartelink) [reply EsBankAccount.Tests/Startup/BankAccountClientTests.fs:16]: There's really no need - this is really just nits and subjective. Good to see you still doing things on this from time to time though (I have other stuff like this almost at standstill which will eventually get bits of time!)

Comment (akhansari) [reply EsBankAccount.Tests/Startup/BankAccountClientTests.fs:16]: Knowledge shared is knowledge squared :)
https://github.com/akhansari/EsBankAccount/pull/4

**Pull request merged**